.github/workflows/test.yml

```diff
@@ -7,25 +7,80 @@ on:
     branches:
       - master
 jobs:
-  build:
-    name: Test, build and lint
+  lint:
+    name: Lint
     runs-on: ${{ matrix.os }}
```

Comment:
While you're here, we're not really making the best us of the matrix strategy, I think we could just set `runs-on: ubuntu-latest` and just preserve `node-version` in the matrix, dropping `os`. What do you think?

Reply:
Sure, I'll update it.